macros/is_incremental_compatible.sql

```diff
@@ -0,0 +1,17 @@
+{% macro is_incremental_compatible() %}
```

Comment:
When I was reviewing Mixpanel, I realized that we provide the [adapter.dispatch](https://github.com/fivetran/dbt_fivetran_log/blob/b82e78e9698666402fb52c488686c079d00ed50b/macros/fivetran_log_lookback.sql#L1-L3) for the incremental lookback window. I realized we should really be doing this for any macro we are adding which has the potential to see macros in other packages which have the same name (which is definitely the case for `is_incremental_compatible`). 

I did some searching on the dbt docs and realized that dbt will by default use the macro defined closest to the place where the macro is being called. For example I tested in a project using both this HubSpot branch and Fivetran Log. Both Fivetran Log and HubSpot used the respective macros appropriately. However, this is not a foolproof solution and could be prone to errors. A more scalable solution would be to alias these macros (eg. hubspot.is_incremental_compatible`) and include the dispatch override functionality. This will give users (and ourselves) the ability to override the macro if needed and ensures dbt will only use the macro defined in the designated project during runtime.

Would you be able to make this update to the macro and then use the package alias when calling it in the respective incremental model blocks.

Reply:
Added the dispatches!